vispy/color/colormap.py

```diff
@@ -440,6 +441,26 @@ def texture_lut(self):
         return texture_LUT
 
 
+class MatplotlibColormap(Colormap):
+    """Use matplotlib colormaps if installed.
+
+    Parameters
+    ----------
+    name : string
+        Name of the colormap.
+    """
+
+    def __init__(self, name):
+        assert has_matplotlib()
+        from matplotlib.cm import ScalarMappable, cmap_d
+        if name not in sorted(cmap_d):
```

Comment:
Is `cmap_d` a dictionary? Is that an advertised/public/documented dictionary that won't be removed any time soon? Wouldn't `if name in cmap_d` work while avoiding the sorting?

Reply:
Hello @djhoese :)

Yes, [cmap_d](https://github.com/matplotlib/matplotlib/blob/3448de363d66d448e853218421f2763bb3508730/lib/matplotlib/cm.py#L20) is a dict with the key the name of the colormap and values are colormap object. I don't think it will be removed soon. I removed the `sorted`.

Vispy also have a [get_colormaps](https://github.com/vispy/vispy/blob/master/vispy/color/colormap.py#L1105) method. If matplotlib is installed, it should probably return the union of vispy and mpl colormaps.